lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.js

```diff
@@ -16,13 +16,17 @@ const defaultApiGatewayLogFormat = [
   'protocol: $context.protocol',
   'responseLength: $context.responseLength',
 ].join(', ');
+const defaultApiGatewayLogLevel = 'INFO';
+const apiGatewayValidLogLevels = ['INFO', 'ERROR'];
```

Comment:
nit: conceptually it'll be nicer to use `Set` here, still then stringification for error message requires to resolve string as `Array.from(apiGatewayValidLogLevels).join(', ')

Reply:
Agreed, it's semantically more correct; I've switched it over.